README.md

```diff
@@ -51,7 +46,7 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.0.0-rc.1
         env:
-          CR_TOKEN: "${{ secrets.CR_TOKEN }}"
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
If I understand correctly, we don't need to users to add the `CR_TOKEN` option at all since GitHub injects `GITHUB_TOKEN` secret also as an environment variable into workflow steps. This LGTM except we should also remove these lines from the readme example workflow too.